src/transacting.md

```diff
@@ -284,12 +284,13 @@ curl -X POST "https://service.centrapay.com/payments/api/transactions.refund" \
 | 194        | 403       | {% break _ INACTIVE_ASSET %}                      | The asset has either expired or been blocked                                                                                                                                                                                                                             |
 | 195        | 400       | {% break - INVALID_ASSET_ID %}                    | The asset corresponding to the asset id is not supported                                                                                                                                                                                                                 |
 | 196        | 400       | {% break - INVALID_WALLET_ADDRESS %}              | The wallet address is not the same as the supported wallet address                                                                                                                                                                                                       |
-| 197        | 400       | {% break - INVALID_TRANSACTION %}                 | The transaction has missing query parameters                                                                                                                                                                                                                             |
+| 197        | 400       | {% break - INVALID_TRANSACTION %}                 | The transaction has either missing query parameters or is not supported                                                                                                                                                                                                  |
```

Comment:
This is not related to partial refunds but is related to a previous PR but I forgot to add this at that time, hence adding it here.